lib/apple_formatter.rb

```diff
@@ -42,7 +42,7 @@ def self.write_strings_file(terms, file, substitutions: nil, print_date: false,
 
         # If plural, use the text for the "one" (singular) entry
```

Comment:
Probably worth updating the comment accordingly 😉

Reply:
Yes, fixed!